lib/builderator/config/rash.rb

```diff
@@ -53,7 +53,7 @@ def merge!(other)
           k = k.gsub(/\-/, '_') if k.is_a?(String)
 
           ## Merge Arrays
-          next self[k] += v if fetch(k, nil).is_a?(Array) && v.is_a?(Array)
+          next self[k] |= v if fetch(k, nil).is_a?(Array) && v.is_a?(Array)
```

Comment:
Is it worth adding some tests around a `Rash` to validate this sort of behavior?